api/who/who.proto

```diff
@@ -28,6 +30,14 @@ message PutLocationRequest {
   string isoCountryCode = 12;
 }
 
+message PutClientSettingsRequest {
+  // Firebase Cloud messaging registration token
+  string token = 1;
+  
+  // User's selected location, ISO 3166-1 alpha-2 (uppercase) code.
+  string isoCountryCode = 12;
```

Comment:
The user may or may not have selected a countrycode; seems simplest to just allow it to be "". Quite likley it can never actually be 'null', I haven't checked the docs.

Reply:
proto3 is reintroducing optional fields but they end up being quite a hassle. Easier to allow "" and then be done with it.

That means fixing the client code to report an empty string when the country isn't set..... or just set country to be a default of empty string.